CONTRIBUTING.md

```diff
@@ -14,6 +14,61 @@ pull request.
 See [GitHub's official documentation](https://help.github.com/articles/using-pull-requests/)
 for more details.
 
+## Developer Setup
+
+Before making changes to markdown files in the handbook, developers should set
+up a [MkDocs development environment](https://www.mkdocs.org/#installation).
+The most consistent way is manually, through Python:
+
+1. Download and [install Python](https://www.python.org/)
+   (3.8 or later is recommended)
+2. (Optional) Create a virtual environment named `mkdocs` in the current
+   directory: `python -m venv mkdocs`
+3. (Optional) Activate the virtual environment by running `activate` in the
+   `Scripts` sub-directory of the `mkdocs` virtual environment
+4. Upgrade pip: `python -m pip install --upgrade pip`
+5. Install dependencies: `pip install -r requirements.txt`
+
+## Previewing Changes to Markdown Files
+
+When making a change to handbook content, use the
+[MkDocs environment](https://www.mkdocs.org/#getting-started) set up in the
+Developer Setup section to start a local testing server:
+
+```bash
+$ mkdocs serve
+INFO    -  Building documentation...
+INFO    -  Cleaning site directory
+INFO    -  Documentation built in 0.58 seconds
+[I 200731 15:48:28 server:334] Serving on http://127.0.0.1:8000
+INFO    -  Serving on http://127.0.0.1:8000
+[I 200731 15:48:28 handlers:62] Start watching changes
+INFO    -  Start watching changes
+[I 200731 15:48:28 handlers:64] Start detecting changes
+INFO    -  Start detecting changes
+```
+
+Then open a browser to <http://127.0.0.1:8000/> to access a preview of the your
+local copy of the handbook.
+
+## Building a Local Copy of the Handbook
+
+The handbook uses MkDocs to produce a static HTML website that can be served by
+any web server. To make the handbook available offline, you can build your own
```

Comment:
In theory, IT could set up a server with a copy of the handbook that then someone else in the company accesses, but since the target audience of the handbook is also IT, I don't think we need to specify that.